cmd/terramate/cli/cli.go

```diff
@@ -200,21 +220,34 @@ func newCLI(
 		}
 	}
 
+	logger := log.With().
+		Str("action", "newCli()").
+		Str("stack", wd).
+		Logger()
+
+	logger.Trace().
+		Msgf("Evaluate symbolic links for %q.", wd)
```

Comment:
```suggestion
		Msg("Evaluate symbolic links for working dir")
```

Rationale: avoid formatting in favor of fields + we already have the working dir value on the logger context (must be fixed from "stack").